packages/forms/src/widgets/DatalistWidget/DatalistWidget.test.js

```diff
@@ -207,6 +207,30 @@ describe('DatalistWidget', () => {
 		expect(toJson(wrapper)).toMatchSnapshot();
 	});
 
+	it('should select known value on input blur', () => {
+		// given
+		const onChange = jest.fn();
+		const wrapper = mount(
+			<DatalistWidget
+				id="myWidget"
+				required
+				schema={schema}
+				onChange={onChange}
+				options={{ restricted: true }}
+			/>
+		);
+		const input = wrapper.find('input').at(0);
+
+		// when
+		input.simulate('focus'); // to display suggestions
+		input.simulate('change', { target: { value: 'banane' } });
+		input.simulate('blur');
+
+		// then
+		expect(onChange).toBeCalled();
+		expect(toJson(wrapper)).toMatchSnapshot();
```

Comment:
useless snapshot here.
find the render of the selected value to ensure the props is 'banane'